### PR TITLE
Finish renaming from Bard to Fili

### PR DESCRIPTION
**Pull request by QubitPi**
https://github.com/yahoo/fili/issues/355

### CODE REVIEW

**Comment by QubitPi** [comment]
A week of search & replace...finally

File: CHANGELOG.md [hunk 1]
```diff
@@ -250,7 +254,7 @@ Removals:
 ### Changed:
 
 - [Prepare for etag Cache](https://github.com/yahoo/fili/pull/289)
-    * Made isOn dynamic on BardFeatureFlag
+    * Made isOn dynamic on FiliFeatureFlag
```
Comment (kevinhinterlong) [inline CHANGELOG.md]: I'm guessing we wouldn't want to refactor this

File: CHANGELOG.md [hunk 2]
```diff
@@ -632,7 +636,7 @@ Changes:
 
 - [Ability to prevent caching of Druid responses larger than the maximum size supported by the cache](https://github.com/yahoo/fili/pull/93)
     * Supported for both Cache v1 and V2
-    * Controlled with `bard__druid_max_response_length_to_cache` setting
+    * Controlled with `fili__druid_max_response_length_to_cache` setting
```
Comment (kevinhinterlong) [inline CHANGELOG.md]: same here

File: CHANGELOG.md [hunk 3]
```diff
@@ -984,7 +988,7 @@ New Capabilities & Enhancements:
     - Case insensitive row and column keys will be deprecated going forward.
     - ***Because this is a breaking change, the fix has been wrapped in a feature flag. For now, this defaults to the
         existing broken behavior, but this will change in a future version, and eventually the fix will be permanent.***
-        - The feature flag for this is `bard__case_sensitive_keys_enabled`
+        - The feature flag for this is `fili__case_sensitive_keys_enabled`
```
Comment (kevinhinterlong) [inline CHANGELOG.md]: and here

**Comment by michael-mclawhorn** [comment]
This needs to wait until a major version change, and then maybe not at all on the package changes.  The config entities, sure, the class names, sure.  But changing every package means every used will have every import broken all at once.  That's some serious pain to give our downstream users.

**Comment by archolewa** [comment]
Please don't change the package name. I will cry. You will see tears. 😭

**Comment by QubitPi** [comment]
@archolewa 

u 1376884462 852911145 fm 214 gp 0

**Comment by michael-mclawhorn** [comment]
I think we should probably not do this.  At least not the package changes.

**Pull request closed without merging**